src/utils/utils.js

```diff
@@ -0,0 +1,20 @@
+function buildDeck(suits, numbers) {
+    return suits.flatMap((suit) => {
+	return numbers.map((number) => {
+	    return {suit: suit, number: number}
+	})
+    })
+}
+
+function shuffleDeck(array) {
+  return array
+    .map((card) => ({ card, value: Math.random() }))
+    .sort((a, b) => a.value - b.value)
+    .map((obj) => obj.card);
+}
+
+function buildRandomDeck(suits, numbers, number_of_cards) {
```

Comment:
No JS a gente usa camel case

```suggestion
function buildRandomDeck(suits, numbers, numberOfCards) {
```